### DISCUSSION TITLE
Faris Masad - repl.it

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#107] [Announcements]
## Background:

[Twitter](https://twitter.com/masadfrost)
[GitHub](https://github.com/masad-frost)
[repl.it](https://replit.com)

## Themes

- Low code/no code setup
- Easier access to education

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin. 

Justin: Hey everyone! Our guest today is 

## Questions

- [x] What is repl.it? How is it different from traditional development?
- [x] repl.it reminds of how we used to do programming, logging into a remote shared machine via terminal. Programming is a wheel, how has this turn of the wheel improved the status quo 
- [x] How does it actually work behind the scenes?
- [x] How does it compare to other tools like codespaces, gitpod, etc?
- [x] Why did you join repl.it as the founding engineer?
- [x] What was the initial scope of repl.it, how has it expanded?
- [x] What are fun and interesting ways people are using repl.it that you didn't expect?
- [x] What were some of the hardest technical challenges of getting repl.it to work?
- [x] What are some cool open source projects that have been made at repl.it? https://github.com/replit/prybar, https://github.com/replit/upm, https://github.com/replit/kaboom, https://github.com/replit/clui
- [x] What's the future of repl.it/What big features have you all been working towards recently?
- [x] What do you think the future of development is? Do you think tools like repl.it will become the default and there will be the "death of localhost"?

## Tooltips

Add them in the comments below!

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. If you want to see the full interview and to be a part of our community subscribe to us on patreon. Thanks for listening!

**Comment by hipstersmoothie** [comment]
My tooltips: 

https://github.com/binwiederhier/replbot
https://pothos-graphql.dev

**Comment by just-be-dev** [comment]
Tooltips:
- [Book] Four Thousand Weeks by Oliver Burkeman (share the fantastic [summary](https://www.mattswain.com/booknotes/four-thousand-weeks) from Matt Swain. 
- [Podcast] https://www.hackingyouradhd.com/

**Comment by masad-frost** [comment]
Ha replbot is cool! I built `evalbot` a few years ago https://github.com/replit/evalbot

My tooltips are
- https://astexplorer.net/
- https://github.com/graphp/graphviz

**Comment by just-be-dev** [comment]
For show notes: https://www.inkandswitch.com/peritext/
